src/ADNLPProblems/triangle.jl

```diff
@@ -0,0 +1,58 @@
+export triangle
+
+function triangle(τ::Float64 = 0.0, x0 = xe, TRIS::Vector{Int64} = Triangles, Const::Vector{Int64} = Constantes; n::Int = default_nvar, type::Val{T} = Val(Float64), kwargs...) where {T}
```

Comment:
You miss a file including `xe`, `Triangles` and `Const`. You can put them in a file `triangle.jl` in a new folder `data` at the same level as `src` and `test`.